tests/test_credential_paths.py

```
@@ -0,0 +1,144 @@
+"""
+Tests for credential path changes to use ~/.zoho-mcp/
+"""
+
+import os
+import tempfile
+from pathlib import Path
+from unittest.mock import patch, MagicMock
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)